### PR TITLE
Use spotless

### PR DESCRIPTION
**Pull request by gnodet**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -93,6 +90,55 @@ limitations under the License.
       </plugins>
     </pluginManagement>
     <plugins>
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
+        <artifactId>spotless-maven-plugin</artifactId>
```
Comment (slawekjaranowski) [inline pom.xml:95]: Why not put in parent?

Comment (gnodet) [reply pom.xml:95]: Yes, that would make sense.  I'll prepare a PR for the parent pom.

File: pom.xml [hunk 2]
```diff
@@ -15,29 +14,27 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
 
   <name>Plexus Common Utilities</name>
   <description>A collection of various utility classes to ease working with strings, files, command lines and
-    more.
-  </description>
+    more.</description>
 
   <scm>
     <connection>scm:git:git@github.com:codehaus-plexus/plexus-utils.git</connection>
     <developerConnection>scm:git:git@github.com:codehaus-plexus/plexus-utils.git</developerConnection>
```
Comment (slawekjaranowski) [inline pom.xml:35]: we need the same protocol like in parent - https in other case links in generated site will be broken

File: pom.xml [hunk 3]
```diff
@@ -15,29 +14,27 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
-    <url>http://github.com/codehaus-plexus/plexus-utils</url>
     <tag>master</tag>
+    <url>http://github.com/codehaus-plexus/plexus-utils</url>
```
Comment (slawekjaranowski) [inline pom.xml:37]: should be like https://github.com/codehaus-plexus/plexus-utils/tree/master to have correct edit button on documentation

**Comment by slachiewicz** [comment]
Done in https://github.com/codehaus-plexus/plexus-utils/commit/968a069aeb3fae38f55e8ec5e420dc4daea99ea9

**Pull request closed without merging**